tech/languages/dotnet/about.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: .NET
+title: .NET Platforms
```

Comment:
I'd prefer `.NET Family`, as suggested by @RheaAyase.

Reply:
@RheaAyase and I agreed on IRC Platforms was good for this.